.golangci.yml

```diff
@@ -0,0 +1,91 @@
+linters-settings:
+#  govet:
+#    check-shadowing: true
+  golint:
+    min-confidence: 0.8
+  gocyclo:
+    min-complexity: 50
+  staticcheck:
+    go: 1.12
+#  maligned:
+#    suggest-new: true
+#  dupl:
+#    threshold: 150
+#  depguard:
+#    list-type: blacklist
+#    include-go-root: false
+#    packages:
+#      - github.com/sirupsen/logrus
+#      - github.com/bmizerany/assert
+#  funlen:
+#    lines: 70
+#    statements: 40
+#  lll:
+#    line-length: 150
+#  nakedret:
+#    max-func-lines: 10
+
+linters:
+  disable-all: true
+  enable:
+#    - asciicheck
+#    - bodyclose
+#    - deadcode
+#    - depguard
```

Comment:
> Go linter that checks if package imports are in a list of acceptable packages

I use this in combination with a blacklist of packages I do not want to be used. Mainly helpful if you e.g. decided to use a certain test package and want to warn developers about using another package.